src/main/resources/assets/reignofnether/lang/en_us.json

```diff
@@ -1,42 +1,677 @@
+    "units.monsters.reignofnether.spider.tooltip2": "Can be upgraded to allow skeletons to ride them.",
+    "units.monsters.reignofnether.spider.tooltip3": "Spiders move much more slowly under sunlight.",
+    "units.monsters.reignofnether.stray": "Stray",
+    "units.monsters.reignofnether.stray.tooltip1": "An chilling variant of the skeleton that fires slowing arrows.",
+    "units.monsters.reignofnether.stray.tooltip2": "Strays will burn under sunlight.",
+    "units.monsters.reignofnether.stray.tooltip3": "Requires an upgrade at the Laboratory.",
+    "units.monsters.reignofnether.warden": "Warden",
+    "units.monsters.reignofnether.warden.tooltip1": "A powerful but blind monster that can fire sonic booms.",
+    "units.monsters.reignofnether.zombie": "Zombie",
+    "units.monsters.reignofnether.zombie.tooltip1": "An undead monster with a basic melee attack.",
+    "units.monsters.reignofnether.zombie.tooltip2": "Zombies will burn under sunlight.",
+    "units.monsters.reignofnether.zombie_villager": "Zombie Villager",
+    "units.monsters.reignofnether.zombie_villager.tooltip1": "An undead worker that can construct and",
+    "units.monsters.reignofnether.zombie_villager.tooltip2": "repair buildings and gather resources.",
+    "units.monsters.reignofnether.zombie_villager.tooltip3": "Zombie villagers DO NOT burn under sunlight.",
+
+    "units.piglins.reignofnether.blaze": "Blaze",
+    "units.piglins.reignofnether.blaze.tooltip1": "A fiery elemental that shoots fireballs from afar.",
+    "units.piglins.reignofnether.blaze.tooltip3": "Requires a Flame Sanctuary.",
+    "units.piglins.reignofnether.blaze.tooltip2": "Attacks are fast and ignite enemies but inaccurate.",
+    "units.piglins.reignofnether.brute": "Brute",
+    "units.piglins.reignofnether.brute.tooltip1": "A piglin armed with a sword for combat.",
+    "units.piglins.reignofnether.ghast": "Ghast",
+    "units.piglins.reignofnether.ghast.tooltip1": "A massive flying creature that fires explosive fireballs",
+    "units.piglins.reignofnether.ghast.tooltip2": "from a great distance to siege units and buildings.",
+    "units.piglins.reignofnether.ghast.tooltip3": "Slow and fragile, but can only be targeted by ranged units.",
+    "units.piglins.reignofnether.ghast.tooltip4": "Requires a Fortress.",
+    "units.piglins.reignofnether.grunt": "Grunt",
+    "units.piglins.reignofnether.grunt.tooltip1": "A piglin worker that can construct and ",
+    "units.piglins.reignofnether.grunt.tooltip2": "repair buildings and gather resources.",
+    "units.piglins.reignofnether.headhunter": "Headhunter",
+    "units.piglins.reignofnether.head_hunter.tooltip1": "A piglin that throws tridents in battle.",
+    "units.piglins.reignofnether.head_hunter.tooltip2": "Requires a Bastion.",
+    "units.piglins.reignofnether.hoglin": "Hoglin",
+    "units.piglins.reignofnether.hoglin.tooltip1": "A beast trained for war. Deals 50%% more damage to buildings.",
```

Comment:
`"units.piglins.reignofnether.hoglin.tooltip1": "A beast trained for war. Deals 50% more damage to buildings.",`